packages/@aws-cdk/cdk/lib/cloudformation/stack.ts

```diff
@@ -22,12 +23,18 @@ export interface StackProps {
    * Optional. If not supplied, the HashedNamingScheme will be used.
    */
   namingScheme?: IAddressingScheme;
+
+  /**
+   * Tags for the Constructs of this stack
+   */
+  tags?: Tags;
 }
 
 /**
  * A root construct which represents a single CloudFormation stack.
  */
-export class Stack extends Construct {
+export class Stack extends Construct implements ITaggable {
```

Comment:
I feel that this is actually not needed both here and in `App`. Practically, every `Construct` may be taggable now, no? So perhaps the usage is: if you want to tag a tree of constructs, just create a `TagManager` yourself. No? Otherwise, we will have to make `Construct` ITaggable...

Reply:
Agree, I thought the concept of just having a `*.tags` object made it easier. However, I agree I'm good to take this out and document the expected behavior.